src/mono/Directory.Build.props

```diff
@@ -47,8 +47,15 @@
     <PlatformConfigPathPart>$(TargetOS).$(Platform).$(Configuration)</PlatformConfigPathPart>
     <RuntimeBinDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsBinDir)', 'mono', '$(TargetOS).$(Platform).$(Configuration)'))</RuntimeBinDir>
     <MonoObjDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsObjDir)', 'mono', '$(PlatformConfigPathPart)'))</MonoObjDir>
-    <MonoLLVMDir Condition="'$(MonoLLVMDir)' == ''">$([MSBuild]::NormalizeDirectory('$(MonoObjDir)', 'llvm'))</MonoLLVMDir>
-    <MonoAOTLLVMDir Condition="'$(MonoAOTLLVMDir)' == ''">$([MSBuild]::NormalizeDirectory('$(MonoObjDir)', 'cross', 'llvm'))</MonoAOTLLVMDir>
+    <MonoAOTEnableLLVM Condition="'$(TargetsiOS)' == 'true'">true</MonoAOTEnableLLVM>
```

Comment:
Why moving these settings from `mono.proj` file?

Reply:
`MonoAOTEnableLLVM` needs to be set to a final value before `_MonoUseAOTLLVMPackage` is defined, and `_MonoUseAOTLLVMPackage` is used by `llvm-init.proj`, which is depended upon by `monoaotcross.proj` and `eng/Subsets.props`. In these cases, `mono.proj` hasn't been evaluated yet.